### PR TITLE
[VarDumper] Minor fix in date caster

### PR DESCRIPTION
**Pull request by maidmaid**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | /
| License       | MIT
| Doc PR        | /

### CODE REVIEW

File: src/Symfony/Component/VarDumper/Caster/DateCaster.php [hunk 1]
```diff
@@ -27,8 +27,8 @@ public static function castDateTime(\DateTimeInterface $d, array $a, Stub $stub,
         $fromNow = (new \DateTime())->diff($d);
 
         $title = $d->format('l, F j, Y')
-            ."\n".$fromNow->format('%R').self::formatInterval($fromNow).' from now'
-            .($location ? ($d->format('I') ? "\nDST On" : "\nDST Off") : '')
+            ."\n".self::formatInterval($fromNow).' from now'
+            ."\n".($location ? ($d->format('I') ? 'DST On' : 'DST Off') : '')
```
Comment (nicolas-grekas) [inline src/Symfony/Component/VarDumper/Caster/DateCaster.php]: this change will add a trailing "\n" when no DST is here - is that desired?

Comment (maidmaid) [inline src/Symfony/Component/VarDumper/Caster/DateCaster.php]: Oops, reverted.

**Comment by nicolas-grekas** [comment]
Fails on Windows

**Comment by maidmaid** [comment]
Fixed.

**Comment by nicolas-grekas** [comment]
Thank you @maidmaid.

**Pull request merged**